openedx/features/course_experience/tests/views/test_course_outline.py

```diff
@@ -34,7 +34,7 @@
 from lms.urls import RESET_COURSE_DEADLINES_NAME
 from openedx.core.djangoapps.course_date_signals.models import SelfPacedRelativeDatesConfig
 from openedx.core.djangoapps.schedules.models import Schedule
-from openedx.core.djangoapps.schedules.tests.factories import ScheduleFactory
+from openedx.core.djangoapps.schedules.tests.factories import ScheduleFactory  # pylint: disable=unused-import
```

Comment:
(copying from other PR)

For all of these disable=unused-import, I think it would be helpful to have a short comment explaining why they're still needed.

Reply:
There is an Issue [BOM-2408](https://openedx.atlassian.net/browse/BOM-2408) in progress right now which will remove such import statements after testing. I didn't remove these in my PR so as to avoid any failure if possible.

Reply:
Should they just be amnesty'd for now so that we can be sure they get addressed later? My concern is that if there's no followup process, developers might think these imports can't be removed, and won't know why.

Reply:
So in BOM-2408 you'll be reviewing all of these markers then? If so, then yeah, that seems fine.